docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -291,6 +291,16 @@ element with not empty text, otherwise the step fails.
 Then elements located `$locator` are sorted by text in $sortingOrder order
 ----
 
+[WARNING]
+====
+The step is deprecated and will be removed in VIVIDUS 0.6.0.
+The replacement pattern is:
+[source,gherkin]
+----
+Then elements located by `$locator` are sorted by text in $sortingOrder order
+----
+====
+
```

Comment:
.Deprecated syntax
[source,gherkin]
----
 Then elements located `$locator` are sorted by text in $sortingOrder order
 ----

[source,gherkin]
----
 Then elements located by `$locator` are sorted by text in $sortingOrder order
 ----